eng/Versions.props

```diff
@@ -116,6 +116,9 @@
     <SystemSecurityCryptographyX509CertificatesTestDataVersion>5.0.0-beta.21159.1</SystemSecurityCryptographyX509CertificatesTestDataVersion>
     <SystemWindowsExtensionsTestDataVersion>5.0.0-beta.21159.1</SystemWindowsExtensionsTestDataVersion>
     <!-- dotnet-optimization dependencies -->
+    <optimizationwindows_ntx64MIBCRuntimeVersion>99.99.99-master-20210317.2</optimizationwindows_ntx64MIBCRuntimeVersion>
```

Comment:
I heard that there is a way to specify version using wildcard, like in versions.prop "<optimizationlinuxx64MIBCRuntimeVersion>99.99.99-master-20210317.2</optimizationlinuxx64MIBCRuntimeVersion>". instruction doc is https://docs.microsoft.com/en-us/nuget/concepts/package-versioning. However, I haven't tried it before. I see using wildcard maybe helpful if later on before preview3 release that we need to update training packages again for some reason and we don't need to updated version in runtime repo. However, I think the risk is quite low as long as you have tested the current training packages.

Reply:
We don't want wildcards here. If we use exact versions, we can tie into the dependency flow.

Reply:
Yes, we whan tie to dependency flow.